.github/CODEOWNERS

```diff
@@ -18,7 +18,7 @@
 ####
 # PRLabel: %App Configuration
 /sdk/appconfiguration/                              @mssfang @alzimmermsft
-/sdk/appconfiguration/spring-cloud-azure-*/         @mrm9084 @saragluna @yiliuTo @chenrujun @zhoufenqin @jialindai
+/sdk/appconfiguration/azure-spring-cloud-azure-*/   @mrm9084 @saragluna @yiliuTo @chenrujun @zhoufenqin @jialindai
```

Comment:
Should it be `azure-spring-cloud-*`?

Reply:
@Zhenlan is correct. The repos are:
/sdk/appconfiguration/azure-spring-cloud-appconfiguration-config-web
/sdk/appconfiguration/azure-spring-cloud-appconfiguration-config
/sdk/appconfiguration/azure-spring-cloud-feature-management-web
/sdk/appconfiguration/azure-spring-cloud-feature-management